metaseq/tasks/streaming_language_modeling.py

```diff
@@ -174,9 +178,15 @@ def _get_sample_prob(self, dataset_lens):
         """
         Get smoothed sampling porbability by corpus. This helps small corpus by upsampling them.
         """
-        prob = dataset_lens / dataset_lens.sum()
-        smoothed_prob = prob**self.args.multicorpus_sampling_alpha
-        smoothed_prob = smoothed_prob / smoothed_prob.sum()
+        if self.args.multicorpus_sampling_maximum == -1:
```

Comment:
Nit: extract out the default value check to a variable, so the two can be coupled (aka define something like `DEFAULT_MULTICORPUS_MAX = -1` somewhere) ? Also what is a reasonable range to set the `multicorpus_sampling_maximum` value?

Reply:
Done. The range would vary depending on the sizes of the datasets being sampled from. Literature has used values like 3000 (We et al., 2021), 65536 etc.